commit 5d14e20e4538153be5c88e83cea7b94e61e0e34d (refs/changes/33/833/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-11-03T14:09:55-07:00 (2 years, 11 months ago)
    
    TOOLS-1599 sdcadm image "version" should include the buildstamp
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/tools/mk-shar b/tools/mk-shar
index 356eda1..fb4e2e8 100755
--- a/tools/mk-shar
+++ b/tools/mk-shar
@@ -177,7 +177,7 @@ cat <<EOM >$OUTMANIFEST
     "uuid": "$(uuid)",
     "owner": "00000000-0000-0000-0000-000000000000",
     "name": "sdcadm",
-    "version": "$VERSION",
+    "version": "$VERSION-$STAMP",
     "description": "Administer SmartDataCenter standups",
     "disabled": false,
     "public": false,
@@ -190,4 +190,3 @@ cat <<EOM >$OUTMANIFEST
 }
 EOM
 echo created $OUTMANIFEST
-
